### PR TITLE
fix: Plank borders and misc styles

### PR DESCRIPTION
**Pull request by richburdon**
- Normalize borders around planks
- Attempt to normalize containers for article/sections
- TODO: Change base colors (less brown)

Screenshot 2024-09-19 at 10 10 44 PM

### CODE REVIEW

File: packages/plugins/plugin-table/src/components/TableContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+// TODO(burdon): Slide
+// <div role='none' className='flex-1 min-bs-0 pli-16 plb-24'>
+
+const TableContainer = ({ role, table }: LayoutContainerProps<Omit<ObjectTableProps, 'role' | 'getScrollElement'>>) => {
+  return (
+    <Table.Root>
+      <Table.Viewport
+        classNames={mx(
+          role === 'article' && 'block-start-[--topbar-size] max-bs-full row-span-2 is-full sticky-top-0 -mt-[1px]',
```
Comment (thure) [inline packages/plugins/plugin-table/src/components/TableContainer.tsx:21]:
```suggestion
          role === 'article' && 'block-start-[--topbar-size] max-bs-full row-span-2 is-full sticky-top-0 -mbs-px',
```

File: packages/ui/react-ui-deck/src/components/Deck/Deck.tsx [hunk 2]
```diff
@@ -169,6 +169,8 @@ const DeckPlankContent = forwardRef<HTMLDivElement, DeckPlankProps>(
         className={mx(
           'grid row-span-3 grid-rows-subgrid group ch-focus-ring-inset-over-all',
           'opacity-0 transition duration-200',
+          // TODO(burdon): Flashes white.
+          !solo && 'border-x _border-red-500 border-separator',
```
Comment (thure) [inline packages/ui/react-ui-deck/src/components/Deck/Deck.tsx:173]:
```suggestion
          !solo && 'border-li border-separator',
```

Comment (thure) [reply packages/ui/react-ui-deck/src/components/Deck/Deck.tsx:173]: See https://stevecochrane.github.io/tailwindcss-logical/#border-width

**Review by thure — APPROVED** [review]
lgtm for now, let’s see how partial opacity on all modals feels, I suspect the novelty may wear off where legibility isn’t as good as it once was

File: packages/ui/react-ui-theme/src/config/tokens/sememes-system.ts [hunk 3]
```diff
@@ -43,13 +44,11 @@ export const systemSememes = {
...
-  modal: {
-    light: ['neutral', surfaceCadence.light[0]],
-    dark: ['neutral', surfaceCadence.dark[5]],
-  },
-  separator: {
-    light: ['neutral', surfaceCadence.light[4]],
-    dark: ['neutral', surfaceCadence.dark[6]],
+  modalSurface: {
+    light: ['neutral', `${surfaceCadence.light[1]}/.88`],
+    dark: ['neutral', `${surfaceCadence.dark[2]}/.88`],
```
Comment (thure) [inline packages/ui/react-ui-theme/src/config/tokens/sememes-system.ts:49]: We’ll likely want to iterate on this, thanks for keeping the previous value.

File: packages/ui/react-ui-deck/src/components/Deck/Deck.tsx (hunk 2 above)
Comment (richburdon) [reply packages/ui/react-ui-deck/src/components/Deck/Deck.tsx:173]: thanks

**Pull request merged**